src/runtime/config/configuration-qemu-se.toml.in

```diff
@@ -0,0 +1,673 @@
+# Copyright (c) 2017-2019 Intel Corporation
+# Copyright (c) 2021 Adobe Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# XXX: WARNING: this file is auto-generated.
+# XXX:
+# XXX: Source file: "@CONFIG_QEMU_IN@"
```

Comment:
Should this point to `CONFIG_QEMU_SE_IN`?

Reply:
Correct! Thanks. :wink: